### ISSUE TITLE
[feature request] add option for printing DB version to STDOUT

### ISSUE THREAD

**Issue by kapsakcj** [ncbi/amr#86]
AFAIK the only way to print the database version is to run an assembly or protein sequence through `amrfinder`, which then prints the db dir and version in the first couple lines.
```
$ amrfinder --plus -n /amrfinder/test_dna.fa -p /amrfinder/test_prot.fa -g /amrfinder/test_prot.gff -O Escherichia >test_both.got
Running: amrfinder --plus -n /amrfinder/test_dna.fa -p /amrfinder/test_prot.fa -g /amrfinder/test_prot.gff -O Escherichia
Software directory: '/amrfinder/'
Software version: 3.10.24
Database directory: '/amrfinder/data/2022-04-04.1'
Database version: 2022-04-04.1
AMRFinder combined translated and protein and mutation search
Running blastp...
Running hmmsearch...
Running blastx...
Running blastn...
Making report...
AMRFinder took 5 seconds to complete
```

It would be an extra cherry on top if a user could provide a flag like so to print the database version to STDOUT:
```
$ amrfinder --db-version
2022-04-04.1
```

Just a thought! Would make for easier tracking of the database version used at runtime.

**Comment by evolarjun** [comment]
Hi Curtis,

I have long wanted to have --version print out the full info:
```
Software directory: '/amrfinder/'
Software version: 3.10.24
Database directory: '/amrfinder/data/2022-04-04.1'
Database version: 2022-04-04.1
```

But I was afraid of breaking peoples automation and docker scripts (including mine). I know it makes parsing the output slightly more complicated, but how would you feel if we added a separate `-V` option to print out the full information above.

Thanks for the suggestion,
Arjun

**Comment by evolarjun** [comment]
Hi @kapsakcj ,

We just released a new version of AMRFinderPlus ([3.10.30](https://github.com/ncbi/amr/releases/tag/amrfinder_v3.10.30)) with a `--database_version` option that prints out the software directory and version as well as the database directory and version.  It's not quite as simple to parse as `-v`, but has more complete info. Hopefully it meets your needs. I'm sure I'll find a use for it in our automated testing and [docker image](https://github.com/ncbi/docker/tree/master/amr) creation as well. Thanks for the suggestion.

Arjun

**Comment by kapsakcj** [comment]
Thanks for implementing this! No worries about the parsing, it will not be difficult at all.
